community/kernel-api/src/main/java/org/neo4j/internal/kernel/api/Token.java

```diff
@@ -47,4 +47,8 @@
     void propertyKeyCreateForName( String propertyKeyName, int id ) throws SomeException;
 
     void relationshipTypeCreateForName( String relationshipTypeName, int id ) throws SomeException;
+
+    int nodeLabel( String name );
+
+    int propertyKey( String name );
```

Comment:
So I guess these are non-creating getters? What if those labels and property keys don't exist? Some javadocs would be nice. Also, `SomeException` needs to be _something else_.